sagemaker/launch/launch_infer.py

```diff
@@ -68,24 +68,19 @@ def run_job(input_args, image, unknownargs):
 
     prefix = f"gs-infer-{graph_name}"
 
+    params = {
+        "graph-data-s3": graph_data_s3,
+        "graph-name": graph_name,
+        "infer-yaml-s3": infer_yaml_s3,
+        "model-artifact-s3": model_artifact_s3,
+        "raw-node-mappings-s3": input_args.raw_node_mappings_s3,
```

Comment:
If we directly add ```raw-node-mappings-s3``` argument here, when it is empty. ```launch/launch_infer.py``` will call ```run/infer_entry.py``` like ```python3 run/infer_entry.py --raw-node-mappings-s3 ``` which will cause an error. Suggest to do what we do here: https://github.com/awslabs/graphstorm/blob/8380f8c7fe35dede91315fa7223eb13baf1d0010/sagemaker/launch/launch_train.py#L72-L75